app/routes/overview/components/utils.tsx

```diff
@@ -40,7 +40,7 @@ export const renderMeta = (
       {item.tags?.length > 0 && (
         <Tags className={styles.tagline}>
           {item.tags.slice(0, 3).map((tag) => (
-            <Tag tag={tag} key={tag} />
+            <Tag tag={tag} key={tag} link={'/articles/?tag=' + tag} />
```

Comment:
Okay, so, I now realized that events _can_ hold tags, despite there being no possible way to actually tag an event in the GUI ... so `item.tags.length` will always be zero. Not sure if we want to check for this here though ... could use `isEvent`